src/components/app/LoginForm.js

```diff
@@ -59,6 +60,19 @@ class LoginForm extends Component {
 
     }
 
+    checkIfAlreadyLogged(err){
+        const {dispatch} = this.props;
+
+        return dispatch(localLoginRequest())
+            .then(response => {
+                if (response.data){
+                    return location.href = '/';
```

Comment:
cause, (imo) the page needs reload. When I used router's push, it threw an error related to redux state updates - "Can only update a mounted or mounting component. This usually means you called setState() on an unmounted component...."
wdyt